test/e2e/storage/persistent_volumes-local.go

```diff
@@ -389,6 +392,7 @@ func setupLocalVolume(config *localTestConfig, node *v1.Node) *localTestVolume {
 	testDirName := "local-volume-test-" + string(uuid.NewUUID())
 	testDir := filepath.Join(containerBase, testDirName)
 	hostDir := filepath.Join(hostBase, testDirName)
+	//populate volume with testFile containing testFileContent
 	writeCmd, _ := createWriteAndReadCmds(testDir, testFile, testFileContent)
 	framework.Logf("Creating local volume on node %q at path %q", node.Name, hostDir)
```

Comment:
This could be `By` instead of Logf -- makes the text stand out more. Personal choice.